jekyll/_cci2/getting-started.md

```diff
@@ -217,7 +217,7 @@ workflows:
             - one
 ```
 
-Read more about workspaces [here](https://circleci.com/docs/2.0/workflows/#using-workspaces-to-share-data-among-jobs).
+Read more about workspaces [here]({{site.baseurl}}/2.0/workflows/#using-workspaces-to-share-data-among-jobs).
```

Comment:
Maybe @rosieyohannan can confirm, but I think if we're going to edit these, we want to avoid using "here" as the link word and re-phrase to link the associated word to the page like "Workspaces"

Reply:
@devinmarieb oh no you're right - I'll update it. Thanks!